c7n/resources/appmesh.py

```diff
@@ -324,3 +327,125 @@ class resource_type(TypeInfo):
             'virtualNodes',
             None,
         )
+
+
+@AppmeshMesh.filter_registry.register('virtual-service')

```

Comment:
I think we can just substitute virtual in the resource name for "appmesh" so we can have these grouped together by the service from a naming perspective.

Reply:
This is fixed to change it to appmesh-service. Please check